serverless/knative_eventing/serverless-pingsource.adoc

```diff
@@ -0,0 +1,33 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-pingsource"]
+= Using PingSource
+:context: serverless-pingsource
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+PingSource is used to periodically send ping events with a constant payload to an event consumer.
```

Comment:
Should this be _A PingSource_? Or _A PingSource object_?
When I first started reviewing (out of published order), i assumed PingSource was a service or an application. 

If so, should you change this throughout?

Reply:
updated to "a PingSource" or "the PingSource" thoughout - it's a type of event source :slightly_smiling_face: